config/locales/en.yml

```diff
@@ -548,10 +550,12 @@ en:
         description: "Recent diary entries from users of OpenStreetMap"
     comments:
       title: "Diary Comments added by %{user}"
-      heading: "%{user}'s Diary Comments"
-      subheading_html: "Diary Comments added by %{user}"
-      no_comments: "No diary comments"
+      heading_html: "%{user}'s Comments"
+      for_diary: "For diary"
+      for_history: "For map edits"
```

Comment:
I don't think the "For" prefix reads very well here and there's also inconsistent pluralisation between the two tabs. I think I would just go with `Diaries` or `Diary entries` for the first one and `Changesets` for the second - we could do `Map edits` but we already using changesets in other places so I think we should be consistent.

Reply:
Changed to "Diary entries" and "Changesets". But user profile menu has "Edits" (plural, changesets not mentioned) and "Diary" (singular, users don't have multiple diaries).